src/client/src/rt-system/ServiceStub.ts

```diff
@@ -40,58 +37,57 @@ export class ServiceStub {
    */
   subscribeToTopic<T>(topic: string, acknowledgementObs?: NextObserver<string>): Observable<T> {
```

Comment:
There is no test associated with subscribeToTopic? I could not find any.

Reply:
That is next